logs/command.go

```diff
@@ -2,104 +2,59 @@ package logs
 
 import (
 	"context"
-	"flag"
-	"fmt"
-	"os"
+	"github.com/urfave/cli/v2"
 )
 
-const logsCommandName = "logs"
-
-type command struct {
-	fs     *flag.FlagSet
-	client *Client
-	opts   *Options
+var flags = []cli.Flag{
+	&cli.StringFlag{Name: "group", Aliases: []string{"g"}, Usage: "group ID to search"},
+	&cli.StringFlag{Name: "min-time", Usage: "earliest time to search from", Required: true},
+	&cli.StringFlag{Name: "max-time", Usage: "latest time to search from", Required: true},
```

Comment:
should these be required?

Reply:
I assumed they were based on some of the parsing code I read, but maybe I misread. I can remove the required flag.

Reply:
Oh yikes.

The default min-time is really quite aggressive.

```
Jun 03 11:22:07 kube-proxy-j6mk9 kube-proxy I0603 11:22:06.435820       1 endpointslicecache.go:358] "Setting endpoints for service port name" portName="swo-k8s-collector/swo-k8s-collector-kube-state-metrics:http" endpoints=["10.224.0.124:8080"]
```

That loads _a lot_ of data--so much so that `less` navigate the amount of data it pulled.

Do we really want to go back that far? Should we have a default min-time? I assume default max-time is now.

Reply:
Yes, setting this to something like 1h ago seems reasonable.

+ if we set these flag as required then examples section should be updated:
```
jakub.skiba@MBKC9NKCG7QM swo-cli % ./swo logs
NAME:
   swo logs - command-line search for SolarWinds Observability log management service

USAGE:
   swo logs [command options]

EXAMPLES:
   swo-cli logs something
   swo-cli logs 1.2.3 Failure
   swo-cli logs -s ns1 "connection refused"
   swo-cli logs "(www OR db) (nginx OR pgsql) -accepted"
   swo-cli logs -g <SWO_GROUP_ID> "(nginx OR pgsql) -accepted"
   swo-cli logs --min-time 'yesterday at noon' --max-time 'today at 4am' -g <SWO_GROUP_ID>
   swo-cli logs -- -redis
```

Reply:
and the binary name too, it' still `swo-cli`

Reply:
Oh snap. Thanks for pointing out the swo-cli usage. Also, just in case that change confused you, it was done at Mark's request. He'd like the CLI to be `swo` instead of `swo-cli`. The binary name is `swo` now.

I'll change the default min-time to `1 hour ago`.